src/oop/programs_with_multiple_classes/hard/q2/MainTest.java

```diff
@@ -0,0 +1,130 @@
+package oop.programs_with_multiple_classes.hard.q2;
+
+import global.ObjectTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static global.tools.CustomAssertions._assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class MainTest {
+    // Java
+
+    private ObjectTest person;
+    private ObjectTest blanket;
+
+    private final String height = "height";
+    private final String size = "size";
+    private final String threadCount = "threadCount";
+
+    private final String personLc = "person";
+    private final String personUc = "Person";
+    private final String blanketLc = "blanket";
+    private final String blanketUc = "Blanket";
+
+    @BeforeEach
+    public void Setup() {
+        String personClassString = "oop.programs_with_multiple_classes.hard.q2." + personUc;
+        String blanketClassString = "oop.programs_with_multiple_classes.hard.q2." + blanketUc;
+        person = new ObjectTest(personClassString);
+        blanket = new ObjectTest(blanketClassString);
+    }
+
+    /**
+     * Person Tests
+     **/
+    @Test
+    public void personClassHasCorrectFields() {
+        String missingFieldMessage = "Your " + personLc + " class is missing a required field.";
+        String incorrectFieldModifierMessage = "A field in your " + personLc + " class has an incorrect modifier.";
```

Comment:
```suggestion
        String incorrectFieldModifierMessage = "A field in your " + personLc + " class has the wrong visibility modifier.";
```

Reply:
What do you think of this?

Reply:
I think that sounds good, good to be specific since we're only checking visibility modifiers here anyways